.github/workflows/cicd-main.yml

```diff
@@ -5347,7 +5345,6 @@ jobs:
       - L2_NeMo_2_Mixtral_Pretraining
       - L2_PTQ_Llama2_INT8_SQ
       - L2_PTQ_Llama2_FP8
-      - L2_NeMo_2_HF_MODEL_IMPORT
```

Comment:
May i ask why this deleted? I see the test above & didnt seem flaky recently

Reply:
Because it's a duplicate of line 5342 (of head)

Reply:
SG